### DISCUSSION TITLE
Supabase copy file from bucket to another using Flutter

### DISCUSSION THREAD

**Discussion by Othman-Sami** [supabase/supabase#21567] [Questions] [answered]
Hi 
I want to ask if its possible to copy file from bucket to another 

I know its could be copy in same bucket 
Like : 
```
await  Supabase.instance.client.storage
            .from("request-images")
            .copy("test.png", "user-images/test.png");
```

But its not allow to copy to another bucket

**Comment by GaryAustin1** [comment, marked as answer]
The storage API does not support this.  It is not a Flutter client limitation.
